packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -1357,31 +1177,19 @@ export namespace EmrCreateCluster {
      * The Amazon EC2 Availability Zone for the cluster. AvailabilityZone is used for uniform instance groups, while AvailabilityZones
      * (plural) is used for instance fleets.
      *
-     * @default EMR selected default
+     * @default - EMR selected default
      */
     readonly availabilityZone?: string;
```

Comment:
What is an EMR selected default?

Reply:
From what I can observe, the locations where we have documented EMR selected default are optional parameters where they have not documented default value or behaviour.
I'm going to open an internal issue with the EMR team to shore up documentation.

For us, we can improve these by using them and observing the behaviour after deploying a stack.
I don't think it's pragmatic at this time and am leaving these as a placeholder in this PR